apps/calendar/js/backend/calendar_service.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+importScripts('../ext/alameda.js');
+
+require.config({
+  baseUrl: '/js/service',
+  paths: {
+    ext: '/js/ext'
+  }
+});
+
+require(['ext/threads'], threads => {
+  var service = threads.service('calendar');
+
+  service.method('start', () => {
+    // TODO
+  });
+
+  service.method('accounts/create', details => {
```

Comment:
all the arguments are causing jshint errors (`details`, `account`, `calendarId`, `event`, `stream`...): https://s3-us-west-2.amazonaws.com/taskcluster-public-artifacts/h39Xe6-LQKy5b4DvNl67Dw/0/public/logs/live_backing.log

we should disable this rule for this file or implement a basic `console.log` on each callback just so the error goes away.